test/docfx.Test/e2e/E2ETest.cs

```diff
@@ -164,7 +164,17 @@ private static TheoryData<string> FindTestSpecs()
 #else
                     var only = false;
 #endif
+                    if (Path.GetFileNameWithoutExtension(file).Contains("localization"))
```

Comment:
Is there another way to determine whether to test from loc? Spec name is prone to change. If someone changes spec name later, it is hard for human or CI to find out that half test cases are cut.